src/main/java/com/sixt/service/framework/rpc/backoff/ExponentialRetryBackOff.java

```diff
@@ -0,0 +1,27 @@
+package com.sixt.service.framework.rpc.backoff;
+
+import java.time.Duration;
+
+public final class ExponentialRetryBackOff
```

Comment:
this class is nearly the same as the one right above.  I would refactor this to be have more clear naming, and for one class to just call the other or have one extend the other